packages/react-search-ui/CHANGELOG.md

```diff
@@ -12,3 +12,18 @@
 ## 0.4.0 (February 11, 2019)
 
 - Search As You Type support via a `searchAsYouType` option on `SearchBox`.
+
+## 0.5.0 (March 6, 2019)
+
+- #90
+- #8
+- #15
+- #107
+- #114
+
+Breaking Changes:
+
+- Generally speaking the data model changed, especially with regards to `filters`, `facets`
```

Comment:
Minor issue: You write `filters`, `facets` and `results` here and in the `packages/react-search-ui-views/CHANGELOG.md`, you write those as Filters, Facets and Results.